rllib/algorithms/crr/torch/crr_torch_policy.py

```diff
@@ -389,14 +407,15 @@ def _compute_critic_loss(
         ).squeeze(-1)
 
         # compute the MSE loss for all q-functions
-        loss_q1 = (target - q1) ** 2
-        loss_q2 = (target - q2) ** 2
-        loss = 0.5 * (loss_q1 + loss_q2)
+        loss_q1 = target - q1
+        loss_q2 = target - q2
+        loss_fn = l2_loss if self.config["td_error_loss_fn"] == "mse" else huber_loss
+        loss = loss_fn(torch.hstack((loss_q1, loss_q2)))
```

Comment:
nit: can you use .stack with (dim=0 or 1).

Reply:
done. I found that it should actually be torch.cat.